notebooks/3_KernelMethods.ipynb

```diff
@@ -1304,10 +1327,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "table_from_dict([krr.statistics(X=X_train, Y=Y_train, K=K_train), \n",
-    "                 krr.statistics(X=X_test, Y=Y_test, K=K_test)], \n",
-    "                 headers = [\"Training\", \"Testing\"], \n",
-    "                 title=\"Kernel Ridge Regression\")"
+    "from utilities.classes import KPCovR # to be replaced with scalers from skcosmo"
```

Comment:
scalers?

Reply:
It should be "to be replaced with the class from skcosmo"